worldengine/a_star.py

```diff
@@ -82,112 +83,110 @@ def _handleNode( self, node, end ):
-                    self.o.pop( i )
-                    self.on.append( n )
-                    self.o.append( n.lid )
+                    self.on.pop(i)
+                    self.o.pop(i)
+                    self.on.append(n)
+                    self.o.append(n.lid)
             else:
                 # new node, append to open list
-                self.on.append( n )
-                self.o.append( n.lid )
+                self.on.append(n)
+                self.o.append(n.lid)
 
         return None
 
-    def findPath( self, fromlocation, tolocation ):
-        self.o = []
-        self.on = []
-        self.c = []
-
-        end = tolocation
-        fnode = self.mh.getNode( fromlocation )
-        if not fnode: # it is possible that fromLocation comes from mountain
+    def find_path(self, from_location, to_location):
+        end = to_location
+        f_node = self.mh.get_node(from_location)
+        if not f_node:  # it is possible that from_location comes from mountain
             return None
-        self.on.append( fnode )
-        self.o.append( fnode.lid )
-        nextNode = fnode
+        self.on.append(f_node)
+        self.o.append(f_node.lid)
+        next_node = f_node
 
         # need to build in a bail-out counter
         counter = 0
 
-        while nextNode is not None:
+        while next_node is not None:
             if counter > 10000:
-                break # no path found under limit
-            finish = self._handleNode( nextNode, end )
+                break  # no path found under limit
+            finish = self._handle_node(next_node, end)
             if finish:
-                return self._tracePath( finish )
-            nextNode = self._getBestOpenNode()
+                return self._trace_path(finish)
+            next_node = self._get_best_open_node()
             counter += 1
 
         return None
 
 
-class SQ_Location:
+class SQLocation:
     """A simple Square Map Location implementation"""
 
-    def __init__( self, x, y ):
+    def __init__(self, x, y):
         self.x = x
         self.y = y
 
-    def __eq__( self, l ):
+    def __eq__(self, l):
         if l.x == self.x and l.y == self.y:
             return 1
         else:
             return 0
 
 
-class SQ_MapHandler:
+class SQMapHandler:
     """A simple Square Map implementation"""
 
-    def __init__( self, mapdata, width, height ):
-        self.m = mapdata
+    def __init__(self, map_data, width, height):
+        self.m = map_data
         self.w = width
         self.h = height
 
-    def getNode( self, location ):
+    def get_node(self, location):
         x = location.x
         y = location.y
         if x < 0 or x >= self.w or y < 0 or y >= self.h:
             return None
-        d = self.m[( y * self.w ) + x]
+        d = self.m[(y * self.w) + x]
 
-#        import constants
-#        if d >= ( constants.BIOME_ELEVATION_MOUNTAIN ): # not over mountains
-#            return None
+        #        import constants
```

Comment:
mmm not sure what we want to do with this. I think the world.elevation['thresholds'] contains the value which tells us at which altitude mountains start